mod/wikirate/set/self/homepage/involved.haml

```diff
@@ -4,9 +4,10 @@
     
   .py-5
     .homepage-involved-cards{ data: {
-      slick: { infinite: false,
+      slick: { infinite: true,
```

Comment:
does the infinite make it go in a continuous loop? Was that requested? I thought before she said she didn't want that.

Reply:
What story does this address?

Reply:
@ethn It addresses task two of this story #184726596. Without infinite = true the focused (purple) slide stops at the 3rd to last slide. Do you have different solution?

Reply:
I'm struggling to connect this "3rd to last slide" comment to the story. Sorry if I'm being thick; it's a long story with lots of comments. Maybe can you show me today?

My concern is that maybe we're trying to address one ux design request by changing other parts of the design. The behavior was specified NOT to be infinite when we implemented it, so we can't just decide to change that without UX (melanie) approval.